java-checks/src/main/java/org/sonar/java/checks/PreferStreamAnyMatchCheck.java

```diff
@@ -21,14 +21,19 @@
 
 import com.google.common.collect.ImmutableSet;
 
+import java.util.ArrayList;
 import org.sonar.check.Rule;
 import org.sonar.java.checks.helpers.MethodsHelper;
 import org.sonar.java.checks.methods.AbstractMethodDetection;
 import org.sonar.java.matcher.MethodMatcher;
 import org.sonar.java.matcher.MethodMatcherCollection;
+import org.sonar.plugins.java.api.tree.ExpressionStatementTree;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonar.plugins.java.api.tree.ExpressionStatementTree'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&branch=SONARJAVA-2429&issues=AWDkm36abBlXo5JPxwyW&open=AWDkm36abBlXo5JPxwyW)

[ISSUE_KEY]: AWDkm36abBlXo5JPxwyW